Docs/source/usage/faq.rst

```diff
@@ -27,6 +27,18 @@ We change the default in ``cmake/dependencies/AMReX.cmake``.
 Note that the tiny profiler adds literally no overhead to the simulation runtime, thus we enable it by default.
 
 
+What do I need to know about using the boosted frame?
+-----------------------------------------------------
+
+The boost transformations are calculated internally to WarpX, so the input deck can be designed in the lab frame and no modification to the physical set-up is needed.
+There are new instabilities that arise (see :ref:`boosted frame theory <theory-boostedframe>`) that require different algorithms for stability; please see the :ref:`boosted examples <usage-examples>` for more help.
+That said, there are some practical items worth considering that we list here to assist in designing and analyzing boosted frame simulations:
+
+- The boosted frame simulation begins at boosted time :math:`t'=0`.
+- Best practice is to set the upper :math:`z` limit of the problem domain to 0 with ``geometry.prob_hi``.   Things moving to the right should start with `z < 0` and things stationary (moving to the left in the boosted frame) should start with `z > 0`.
```

Comment:
Actually, only one thing, could you maybe use the `:math:` environment here too?
```suggestion
- Best practice is to set the upper :math:`z` limit of the problem domain to 0 with ``geometry.prob_hi``.   Things moving to the right should start with :math:`z < 0` and things stationary (moving to the left in the boosted frame) should start with :math:`z > 0`.
```